doc/articles/faq.md

```diff
@@ -2,39 +2,39 @@
 uid: Uno.Development.FAQ
 ---
 
-# FAQ: General
+# Frequently Asked Questions
```

Comment:
Do we wanna stick to the plain Frequently Asked Questions there? I have made a change to :General recently after a discussion with Sasha as it makes all the FAQs look like this in the search with all the FAQs in the docs.
![image](https://github.com/unoplatform/uno/assets/78549750/94e4b215-02a2-412a-a48e-ea0a6b5efc00)

Perhaps if we change the others, this can stay plain, what do you think?